code.go

```diff
@@ -32,6 +32,14 @@ var _ flag.Value = (*Code)(nil)
 // Assert that Code implements fmt.Stringer without a pointer.
 var _ fmt.Stringer = Code(0)
 
+// ReservedStart is the (inclusive) start of the reserved range of codes that
+// are safe to use for internal purposes.
+const ReservedStart = Code(0x300000)
+
+// ReservedEnd is the (exclusive) end of the reserved range of codes that are
```

Comment:
I think this is inclusive, FYI. I'd imagine 0x400000 would be exclusive.

If you are right, then the upstream spec is misleading, because it just says:

> Range: 0x300000 – 0x3FFFFF

Reply:
doh! thanks for picking that up, here I was being all clever with my clear wording.